src/services/sendEmail.js

```diff
@@ -1,10 +1,27 @@
 // using Twilio SendGrid's v3 Node.js Library
 // https://github.com/sendgrid/sendgrid-nodejs
+const { UserInputError } = require('apollo-server-errors');
 const sgMail = require('@sendgrid/mail');
-const { SENDGRID_API_KEY } = require('../config/config');
+const { SENDGRID_API_KEY, CLIENT_URL } = require('../config/config');
+const models = require('../models');
 
 sgMail.setApiKey(SENDGRID_API_KEY);
 
+// TO DO: Edit inivitation email subject, html, and generate email verification link
+const createVerificationEmail = hash => {
+  if (hash == null) {
+    throw new Error('Null Hash');
+  }
+
+  const inviteUrl = `${CLIENT_URL}/verify/${hash}`;
+
+  return {
+    from: 'hongyichen@uwblueprint.org',
+    subject: `Verify Email for Building Up`,
+    html: `Thank you for signing up with <strong>Building Up</strong>. Please verify your email using this link: <a href="${inviteUrl}">${inviteUrl}</a>`,
```

Comment:
A reasonable subject might be: `Verify your email for Raising the Roof's Toque Campaign`

A reasonable body might be: `Thank you for signing up! Please verify your email with this link: {link}`

Reply:
https://uwblueprint.slack.com/archives/C01PVF15KJR/p1618186703002300

Reply:
Oops definitely my bad! I'll make that fix.